test/Test_Elaboration.re

```diff
@@ -3,7 +3,25 @@ open Language;
 
 /*Create a testable type for dhexp which requires
   an equal function (dhexp_eq) and a print function (dhexp_print) */
-let dhexp_typ = testable(Fmt.using(Exp.show, Fmt.string), DHExp.fast_equal);
+let dhexp_typ =
+  testable(
+    Fmt.using(Exp.show, Fmt.string),
+    // This is syntactic with ignore_wrappers=true
+    Equality.equality(
+      ~type_alpha=false,
+      ~exp_alpha=false,
+      ~ignore_wrappers=true,
```

Comment:
This is being done because we have parens in the expectations for the post-elaboration code. We can remove parens speculatively but we'd either have to write the ast in ocaml syntax or make sure the expression doesn't need it.